### DISCUSSION TITLE
Chat Bubble Color chat-bubble-base-200

### DISCUSSION THREAD

**Discussion by mkasberg** [saadeghi/daisyui#3299] [Ideas / feature requests]
I was working with DaisyUI chat bubbles recently and I was a little surprised to find that it isn't currently possible (I believe) to do `chat-bubble-base-200`. Looks like (if I'm reading the source correctly) the default is neutral, and it supports all the other colors (primary, accent, etc). In my example, I would have been trying to experiment with a base-200 or base-300 chat bubble on a base-100 background.

https://github.com/saadeghi/daisyui/blob/3ba8deac5acc65c9c0233f60c681acd773befe33/src/components/styled/chat.css#L7-L10

Is there some reason we wouldn't want to do this in the design system?

If it would be reasonable to add this, it seems like it would be easy to add support for `chat-bubble-base-200` (and `chat-bubble-base-300` and `chat-bubble-base-100`). If you're open to it I'd be happy to open an issue and also perhaps try tackling this change myself. (Also happy to discuss here whether this is even a good idea or not!)

**Comment by saadeghi** [comment]
This will be added in daisyUI 5.
